dlc/src/main/scala/org/bitcoins/dlc/testgen/DLCTestUtil.scala

```diff
@@ -67,7 +67,7 @@ object DLCTestUtil {
     val func = DLCPayoutCurve(
       Vector(
         OutcomePayoutEndpoint(0, leftVal),
-        OutcomePayoutEndpoint(botCollar, leftVal),
+        OutcomePayoutEndpoint(botCollar + 1, leftVal),
```

Comment:
I assume this fix made tests pass in the `testUntilFailed` command?

Reply:
Yes